web_development_101/project_html_css.md

```diff
@@ -1147,6 +1147,8 @@ There are a couple of ways to go about doing this, but the simplest is this:
 * [Akshat Dubey's Solution (Easy)](https://github.com/akshatdb/google-homepage) | [View in browser](https://akshatdb.github.io/google-homepage/)
 * [Siddharth Jaiswal's Solution(Easy)](https://github.com/Sidd0602/google-homepage/) | [View in browser](https://sidd0602.github.io/google-homepage/)
 
```

Comment:
Looks this there is an extra newline here, can you remove it please @duddhoen92?